### PR TITLE
add unstack paddle frontend manipulation

### LINKED ISSUE #19235: unstack

### PR DESCRIPTION
**Pull request by NripeshN**
Close #19235

### CODE REVIEW

**Comment by NripeshN** [comment]
add_frontend_checklist

File: ivy/functional/frontends/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -96,3 +96,12 @@ def cast(x, dtype):
 @to_ivy_arrays_and_back
 def broadcast_to(x, shape, name=None):
     return ivy.broadcast_to(x, shape)
+
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("bool", "float32", "float64", "int32", "int64")},
```
Comment (Aarsh2001) [inline ivy/functional/frontends/paddle/tensor/manipulation.py:102]: `bool` is not registered as a dtype for the `_unstack` kernel in the paddle backend, and so if you see the logs you'll see a runtime error. Can you please remove `bool` from the list and run the test again ?

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]

**Comment by NripeshN** [comment]
Hi @Aarsh2001, Could you please review this PR.

**Review by Aarsh2001 — APPROVED** [review]
My apologies for the delay ! 😅 LGTM !

**Pull request merged**